src/frontends/pytorch/src/op_table.cpp

```diff
@@ -125,6 +125,7 @@ OP_CONVERTER(translate_lstm);
 OP_CONVERTER(translate_masked_fill);
 OP_CONVERTER(translate_masked_scatter);
 OP_CONVERTER(translate_max);
+OP_CONVERTER(translate_mv);
```

Comment:
Please also remove

Reply:
why to remove it

Reply:
there is no need in `translate_mv` function if you use generic helper, like `translate_1to1_match_2_inputs`